cmd/up.go

```diff
@@ -29,9 +29,21 @@ func UpCommand() cli.Command {
 			Usage: "Specify an alternate project name (default: directory name)",
 		},
 	}...)
+
+	cmd.Action = app.WithProject(factory, ProjectUp)
```

Comment:
I don't really know this codebase well, but having to set the Action here kind of scares me. Seems like this bit of code shouldnt need changed. What's the reason for changing it?